pages/blog/miniseries-part_1.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Mini series - Speed up your development process (part 1)"
```

Comment:
```suggestion
title: "Speed up your development process (part 1)"
```

I don't think title must mention that it is some kind of miniseries, part1 already indicates there will be more

Reply:
Yea I agree 👍